services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/muutosilmoitus/MuutosilmoitusController.kt

```diff
@@ -40,7 +52,7 @@ class MuutosilmoitusController(private val muutosilmoitusService: Muutosilmoitus
     @ApiResponses(
         value =
             [
-                ApiResponse(description = "The created täydennys", responseCode = "200"),
+                ApiResponse(description = "The created muutosilmoitus", responseCode = "200"),
```

Comment:
Is the text "and there needs to be an open information request for the application" unnecessary for this controller (a copy-paste from täydennys-controller?)?

Reply:
Thank you, this was a copy-paste mistake.